src/scss/pages/_learn-paths.scss

```diff
@@ -0,0 +1,34 @@
+@media (min-width: 865px) {
+  .learn__path-hero {
+    height: auto;
+    max-width: initial;
+    width: 420px;
+  }
+}
+
+.learn-header {
```

Comment:
If you use BEM notation (.learn__path-hero) stay with BEM (so .learn-header and .learn-link are incorrect)